bot/plugins/instance.py

```diff
@@ -210,6 +209,46 @@ async def on_component_interaction(event: hikari.InteractionCreateEvent) -> None
     await inst.execute()
 
 
+def create_input_args_modal(
+    event: hikari.InteractionCreateEvent,
+    msg_instance: Instance,
+):
```

Comment:
This needs a return type. `list[...]`. Whatever type `build_modal_action_row` returns

Reply:
Fixed